resources/app/Info.plist

```diff
@@ -88,6 +88,29 @@
         </array>
       </dict>
     </dict>
+    <dict>
+      <key>UTTypeIdentifier</key>
+      <string>org.cryptomator.encrypted-cryptomator-vault-file</string>
+      <key>UTTypeConformsTo</key>
+      <array>
+        <string>public.data</string>
+      </array>
+      <key>UTTypeDescription</key>
+      <string>Encrypted Cryptomator vault file</string>
+      <key>UTTypeIconFile</key>
+      <string>Cryptomator-Vault.icns</string>
+      <key>UTTypeTagSpecification</key>
+      <dict>
+        <key>public.filename-extension</key>
+        <array>
+          <string>c9r</string>
```

Comment:
```suggestion
          <string>c9r</string>
          <string>c9s</string>
```

Reply:
GitHub's UI wrongly shows this suggestion to be ignored, when it was indeed added in the commit f5b7007e58698f2c0fe254df331905bc10c2ae0f.